BDArmory/Radar/VesselRadarData.cs

```diff
@@ -1162,14 +1162,20 @@ private void WindowRadar(int windowID)
                 GUI.matrix = Matrix4x4.identity;
             }
 
-            if (noData && iCount == 0)
+            if (noData)// && iCount == 0)
             {
-                GUI.Label(RadarDisplayRect, "NO DATA\n", lockStyle);
+                if (iCount > 0)
+                    DrawDisplayedIRContacts();
+                else
+                    GUI.Label(RadarDisplayRect, "NO DATA\n", lockStyle);
             }
             else
             {
                 DrawDisplayedContacts();
+                if (iCount > 0)
+                    DrawDisplayedIRContacts(); //something is throwing a GUIClips NRE somewhere before DisplayRadarControls()
```

Comment:
That's probably the line
```csharp
if (availableRadars.Count + availableIRSTs.Count == 0) return;
```
right after `GUI.BeginGroup(RadarDisplayRect);` due to the unbalanced `Begin`...`End`.

Reply:
Ah, yeah, that would do it.